examples/raspberrypi3.toml

```diff
@@ -0,0 +1,115 @@
+#
+# Pin Map for Raspberry Pi 3
```

Comment:
Are there differences in GPIOs for RPi2/3?  I know he 2/3 have more IOs than the RPi1, but I think 2 and 3 may have the same pinout.